### PR TITLE
JDA 5 alpha 6 changes

### PR DESCRIPTION
**Pull request by MinnDevelopment**

### CODE REVIEW

**Comment by Speenah** [comment]
Could you include making `CoroutineEventManager` `open` with these changes? I'd like to manage MDC context from one place and also have it propagated to listeners created through the extension functions. I can put up a PR but since it's a really small change didn't think it was worth it

**Comment by MinnDevelopment** [comment]
@Speenah done, you can use the commit right away if you like.

**Comment by Speenah** [comment]
Awesome, thanks!

File: build.gradle.kts [hunk 1]
```diff
@@ -28,7 +28,8 @@ repositories {
 }
 
 dependencies {
-    compileOnly("net.dv8tion:JDA:5.0.0-alpha.5")
+//    compileOnly("net.dv8tion:JDA:5.0.0-alpha.5")
+    compileOnly("com.github.dv8fromtheworld:JDA:a77218c")
```
Comment (MinnDevelopment) [inline build.gradle.kts:32]:
```suggestion
    compileOnly("net.dv8tion:JDA:5.0.0-alpha.9")
```

File: build.gradle.kts [hunk 2]
```diff
@@ -9,7 +9,7 @@ plugins {
 group = "dev.minn"
 version = "0.8.4-alpha.5"
```
Comment (MinnDevelopment) [inline build.gradle.kts:10]:
```suggestion
version = "0.8.5-alpha.9"
```

**Pull request merged**